### PR TITLE
sync signalr test scenarios

### PR DESCRIPTION
**Pull request by zhenglaizhang**
Syncing existing SignalR test scenarios from ADO repo.

### CODE REVIEW

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2020-07-01-preview/scenarios/peArmTemplate/template.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
+{
```
Comment (bjqian) [inline specification/signalr/resource-manager/Microsoft.SignalRService/preview/2020-07-01-preview/scenarios/peArmTemplate/template.json:1]: Isn't this file deleted in the last commit?

**Comment by zhenglaizhang** [comment]
Validation errors are due to wrong swagger detection logic, I am fixing it.
/cc @changlong-liu for awareness if want to try SDK gen in newer API version and more examples coveraged

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2020-07-01-preview/scenarios/peArmTemplate/template.json (hunk 1 above)
Comment (zhenglaizhang) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2020-07-01-preview/scenarios/peArmTemplate/template.json]: seems not, just removed it, and also remove it from the GitHub PR

**Pull request closed without merging**